python/triton_kernels/triton_kernels/tensor_details/layout_details/blackwell_scale.py

```diff
@@ -58,6 +58,43 @@ def swizzle_block_shape(self, block_shape):
         return [1, block_shape[0] // 128, block_shape[1] // 4, 2, 256]
 
 
+class BlackwellActMXScaleLayout(Layout):
+    # Swizzling for activation tensor [M, K], M can be ragged dimension and equals to sum of expert bs
+    name: str = "BLACKWELL_SCALE"
+
+    def __init__(self, shape, ex_hist: list[int] | None = None) -> None:
+        super().__init__(shape)
+        assert len(shape) == 3, f"Only support 3D shape for BlackwellActMXScaleLayout, got {shape}"
+        (
+            *self.leading_shape,
+            self.M,  # sum of expert bs
+            self.K,
+        ) = shape
+        self.B = math.prod(self.leading_shape)
+        self.mode = "batched"
+        self.ALIGN_K = 8
+        self.ALIGN_M = 128
+        self.SWIZZLE_K = 4
+        self.K_pad = (self.K + self.ALIGN_K - 1) // self.ALIGN_K * self.ALIGN_K  # min multiple of ALIGN_K
+        self.M_pad = (self.M + self.ALIGN_M - 1) // self.ALIGN_M * self.ALIGN_M
```

Comment:
For grouped gemm, I expected that padding needs to be done per group. This looks no different from the way padding is done for a non grouped-gemm matmul in https://github.com/triton-lang/triton/blob/main/python/test/unit/language/test_matmul.py#L761-L763.

If this padding is supposed to be correct, what am I missing?